snipsnapdev/snipsnap
# feat: improve editor syntax highlight

**Improve editor syntax highlight**

Some examples:
1. Add Golang syntax support.
2. Add Ruby syntax support

**Steps to test**

1. Open a new template in Snipsnap Editor
2. Try to create `*.go` and `*.rb` files, check syntax highlighting.

**Screenshots**

<img width="1152" alt="Screenshot 2021-06-18 at 18 04 26" src="https://user-images.githubusercontent.com/1070568/122581407-b01b9400-d05f-11eb-817d-d6b8b9c61429.png">
<img width="1168" alt="Screenshot 2021-06-18 at 18 04 21" src="https://user-images.githubusercontent.com/1070568/122581430-b578de80-d05f-11eb-8f08-f60e3a473867.png">
